src/main/java/net/playeranalytics/extension/quests/QuestsExtension.java

```diff
@@ -72,50 +73,106 @@ public CallEvents[] callExtensionMethodsOn() {
         };
     }
 
+    @StringProvider(
+            text = "Quest Points",
+            description = "Total amount of Quest Points",
+            priority = 100,
+            iconName = "certificate",
+            iconColor = Color.AMBER
+    )
+    @Tab("Statistics")
+    public String points(UUID playerUUID) {
+        try {
+            return Optional.of(String.valueOf(getQuester(playerUUID).getQuestPoints())).orElse("0");
```

Comment:
Assuming getQuestPoints() can return null, this optional needs to use Optional.ofNullable directly on the return value. Suggestion assumes that the method is changed to `@NumberProvider`
```suggestion
            return Optional.ofNullable(getQuester(playerUUID).getQuestPoints()).map(Long::valueOf).orElse(0L);
```

Reply:
Does not return null